src/popup.ts

```diff
@@ -726,10 +726,11 @@ async function runThePopup(messageListener?: MessageListener): Promise<void> {
      * @param {float} seconds 
      * @returns {string}
      */
-    function getFormattedHours(minues) {
-        const hours = Math.floor(minues / 60);
-        return (hours > 0 ? hours + "h " : "") + (minues % 60).toFixed(1);
-    }
+	function getFormattedHours(minutes) {
+		minutes = Math.round(minutes * 10) / 10
```

Comment:
I think this should be floor

Reply:
Not necessarily. If there would be floor saved time will be less accurate. 
For example if saved time would be 119.9 with floor it will result in 1h 59min but with round it will be 2h 0min

Reply:
It's really up to you how whould you like to handle this.

Reply:
Ah yea, that works. I forgot this is happening before it converts to hours